src/crepe/ComponentManager.h

```diff
@@ -137,6 +146,9 @@ class ComponentManager {
 	std::unordered_map<std::type_index, std::vector<std::vector<std::unique_ptr<Component>>>>
 		components;
 
+	//! Persistent flag for each GameObject
+	std::vector<bool> persistent = {false};
```

Comment:
Omdat we toch alleen random access nodig hebben kan dit beter een `std::unordered_map<game_object_id_t, bool>` zijn denk ik. Dan hoef je ook niet steeds `.resize()` te gebruiken.